src/drivers/imx/sdma.c

```diff
@@ -208,6 +208,8 @@ static void sdma_init_c0(struct dma *dma)
 	pdata->ccb = sdma_pdata->ccb_array;
 	pdata->hw_event = -1;
 	dma_chan_set_data(c0, pdata);
+
+	dma_reg_write(dma, SDMA_CHNPRI(0), SDMA_MAXPRI);
```

Comment:
I think commit title should specify that it's SDMA on this one as well?

Reply:
sorry, i don't get what you mean.

Reply:
You just mentioned drivers: imx: without the "sdma:" prefix on this last commit.

Reply:
I see. Thanks. Will fix.